### PR TITLE
Fucking Bood

### PR DESCRIPTION
**Pull request by JamieD1**
Fucking Bood

#### Changelog

:cl:    
bugfix: Fucking Bood
/:cl:

### CODE REVIEW

**Review by SomeguyManperson — CHANGES_REQUESTED** [review]
Icons shouldnbe moved to the modular yogs food icon file for a better fix

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 1]
```diff
@@ -97,11 +97,12 @@
 /obj/item/reagent_containers/food/snacks/donut/jelly/laugh
 	name = "sweet pea jelly donut"
 	desc = "Goes great with a glass of Bastion Burbon!"
+	icon = icons/obj/food/food.dmi
```
Comment (jcatjmeow) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]:
```suggestion
	icon = 'icons/obj/food/food.dmi'
```

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 2]
```diff
@@ -114,6 +115,7 @@
 /obj/item/reagent_containers/food/snacks/donut/jelly/slimejelly/laugh
 	name = "sweet pea jelly donut"
 	desc = "Goes great with a glass of Bastion Burbon!"
+	icon = icons/obj/food/food.dmi
```
Comment (jcatjmeow) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]:
```suggestion
	icon = 'icons/obj/food/food.dmi'
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]
Icons are stored as strings

**Review by SomeguyManperson — APPROVED** [review]
should work

**Pull request merged**